Dockerfile

```diff
@@ -1,7 +1,7 @@
 ############################################################
 # Build stage
 ############################################################
-FROM node:18-alpine AS build
+FROM node:lts-alpine AS build
```

Comment:
Can switch back to `its` as `linux/amd64` and `linux/arm64/v8` seem to not have networking issues during the QEMU build